### PR TITLE
Changes over icon-font path

### PR DESCRIPTION
**Pull request by xavijam**
Plus, fix deep-linking in styleguide.

Take a look @javierarce please.

### CODE REVIEW

File: styleguide/_template.html [hunk 1]
```diff
@@ -10,12 +10,20 @@
...
+          }, 0);
+        }
+
         // Show section in Styleguide
         j('#styleguide .section[data-section="' + j('#styleguide-menu a').first().data('section') + '"]').show();
 
         // Click on link with subheadings
         j('#styleguide-menu .submenulink').click(function(event) {
-            event.preventDefault();
+            // event.preventDefault();
```
Comment (javierarce) [inline styleguide/_template.html]: Could we remove this comment?

File: styleguide/_template.html [hunk 2]
```diff
@@ -28,7 +36,7 @@
 
         // Click on link without subheadings
         j('#styleguide-menu .sectionlink').click(function(event) {
-            event.preventDefault();
+            // event.preventDefault();
```
Comment (javierarce) [inline styleguide/_template.html]: Ditto.

**Comment by xavijam** [comment]
Done :)

**Comment by javierarce** [comment]
Great! Looks good to me :+1:

**Pull request merged**